bumble/rfcomm.py

```diff
@@ -609,6 +629,19 @@ def connect(self) -> None:
         self.connection_result = asyncio.get_running_loop().create_future()
         self.send_frame(RFCOMM_Frame.sabm(c_r=self.c_r, dlci=self.dlci))
 
+    async def disconnect(self) -> None:
```

Comment:
Shall we cover new functions in rfcomm_test?

Reply:
Yes. Done.